bibliopixel/remote/control.py

```diff
@@ -34,10 +34,20 @@ def __init__(self, layout, animations,
                  external_access=False, port=5000,
                  title='BiblioPixel Remote', bgcolor='black',
                  font_color='white', default=None,
-                 triggers=[]):
+                 triggers=[], auto_demo=None):
 
         self.internal_delay = 0  # never wait
 
+        if auto_demo:
```

Comment:
But why is auto demo specific to remote?

Why wouldn't you want to apply it to, say, an existing sequence with no web interface?

Basically, we are generating all these very specific parts which interoperate with nothing.  It's not a good future-proof design.

For example, I have in a separate project something that controls BP from MIDI.  So far I've been able to reuse almost nothing from the whole Remote system, because it's tightly coupled with everything it uses - http, flask, even the interface.

And as you put more features into it, your chances of reusing code from this drop even further.

In my heart, I see Remote as a dead-end that will be thrown away eventually, because I can't see anything I can do with it aside from exactly this one thing - "a list of rectangular areas on an HTML page that click and start animations".  That isn't a good feeling to have about code that is weeks old...